app/views/evaluators/index.html.erb

```diff
@@ -151,3 +163,15 @@
     <%= render 'delete_evaluator_modal' %>
   </div>
 </div>
+  <div class="usa-alert usa-alert--info width-tablet margin-bottom-5">
```

Comment:
don't show this if there are no current evaluators (just show the "No Evaluators" alert)